integrations/posthog/gitbook-manifest.yaml

```diff
@@ -0,0 +1,50 @@
+name: posthog
+title: PostHog
+icon: ./assets/icon.png
+previewImages:
+    - ./assets/posthog-preview.png
+    - ./assets/posthog-preview-2.png
+description: Plug your GitBook site to your PostHog installation.
+externalLinks:
+    - label: Documentation
+      url: https://www.gitbook.com/integrations/posthog
+visibility: public
+script: ./src/index.ts
+# The following scope(s) are available only to GitBook Staff
+# See https://developer.gitbook.com/integrations/configurations#scopes
+scopes:
+    - space:script:inject
+organization: d8f63b60-89ae-11e7-8574-5927d48c4877
+contentSecurityPolicy:
+    script-src: https://eu.posthog.com https://us.posthog.com;
+summary: |
+    # Overview
+    This integration allows to add the PostHog installation snippet on your published GitBook site.
+
+    # How it works
+    The integration injects the PostHog script on your page, using the configured project API key and instance address, 
+    so that you can get analytics information from your GitBook site directly inside of PostHog.
+
+    # Configure
+    Install the integration on the GitBook space of your choice.
+    Locate the PostHog project API key and instance address you are using, which is available in PostHog's installation section.
+
+categories:
+    - analytics
+configurations:
+    space:
+        properties:
+            projectApiKey:
+                type: string
+                title: Project API Key
+                description: Available in the PostHog installation snippet, as the first parameter, starting with 'phc_'
+            instanceAddress:
+                type: string
+                title: Instance address
+                description: Depends on your PostHog installation's location (either EU, or US)
+                enum:
+                    - https://eu.posthog.com
+                    - https://us.posthog.com
```

Comment:
I feel using `US` and `EU` as enum values is a better user experience as looking at an url can be abit meh in that select dropdown.

So basically you capture `instanceLocation` (US/EU/etc) instead of `instanceAddress` and in your integrations code you obtain the instance address through a mapping

```js
const instanceAddresses = {
  us: 'https://us.posthog.com',
  eu: 'https://eu.posthog.com',
}

const instanceLocation = environment.spaceInstallation.configuration.instanceLocation;
const instanceAddress = instanceAddresses[instanceLocation.toLowerCase()];
```

Reply:
Right that's a better look and feel indeed. Let's do that instead